redis/index.js

```diff
@@ -10,6 +10,10 @@ client.on('error', err => {
   console.log('error', err)
 })
 
+
+//this method is no longer applicable with user tiering. Reset periods are now determined by the api key registration date, and calculating 30 day intervals based on that initial generation date
```

Comment:
if it's no longer applicable, delete it. future devs don't care what it used to be, just what it is and does.